sections/chapters/conclusion.tex

```diff
@@ -1,7 +1,95 @@
 \chapter{Conclusion}\label{ch:conclusion}
 
 \section{Answering The Problem Statement}
```

Comment:
What I meant by "answering the problem statement" is actually rewriting or referencing the questions in the problem statement and answering them one by one. This could also work, I don't know if AAU has any hard requirements about it, but we need to change the section title (e.g. "reflection").

Reply:
which question did I miss?

Reply:
Maybe this one:
• What are the existing tools in the industry capable of?

But what I mean is, UCN wanted us to be very explicit about how we answer the problem statement. As in, list the questions again, but this time with an answer for each. Again, I don't know if this is a hard requirement at AAU, we can ask Ben

Reply:
Sure, I don't think it matters very much, but I'll send Ben a question